File: atcoder.jp/contests/code-festival-2016-qualb/tasks/codefestival_2016_qualB_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

vector<pair<char, char>> zip(const string &a, const string &b) {
  size_t n = min(a.size(), b.size());
  vector<pair<char, char>> ret(n);
  for (size_t i = 0; i < n; i++) ret[i] = {a[i], b[i]};
  return ret;
}

int main() {
  string S;
  cin >> S;

  int ans = 0;
  for (auto &&[ch1, ch2] : zip(S, "CODEFESTIVAL2016")) ans += ch1 != ch2;

  cout << ans << endl;
}

File: atcoder.jp/contests/code-festival-2016-qualb/tasks/codefestival_2016_qualB_a/main.go
package main

import (
	"fmt"
)

func main() {
	var S string
	fmt.Scanf("%s", &S)

	T := "CODEFESTIVAL2016"

	ans := 0
	for i := 0; i < 16; i++ {
		if S[i] != T[i] {
			ans++
		}
	}

	fmt.Println(ans)
}
